types/glob/index.d.ts

```diff
@@ -56,6 +56,7 @@ declare namespace G {
         nonegate?: boolean | undefined;
         nocomment?: boolean | undefined;
         absolute?: boolean | undefined;
+        allowWindowsEscape?: boolean | undefined;
```

Comment:
please bump the version in the header, otherwise lgtm

Reply:
Done. TY!